docs/network-policy.md

```diff
@@ -0,0 +1,70 @@
+# NetworkPolicy
+Kubernetes NetworkPolicy documentation: https://kubernetes.io/docs/concepts/services-networking/network-policies/#behavior-of-to-and-from-selectors
+
+Each NetworkPolicy object consists of three sections:
+
+1. `podSelector`: a label selector that determines which pods the NetworkPolicy applies to
+2. `ingress`: determines which entities ingress traffic can be received from by the pods selected by the NetworkPolicy
+3. `egress`: determines which entities egress traffic can be sent to by pods selected by the NetworkPolicy
+
+## Unicast default-deny
+When a pod is selected by one or more NetworkPolicies it becomes isolated and all unicast ingress and egress traffic is blocked. Two global PortGroups are used to facilitate this behavior: `ingressDefaultDeny` and `egressDefaultDeny`.  Any pod selected by a NetworkPolicy in any namespace is added to these PortGroups.
```

Comment:
"When a pod is selected by one or more NetworkPolicies that affect ingress, it becomes isolated for ingress, and all unicast ingress traffic is blocked. When a pod is selected by one or more NetworkPolicies that affect egress, it becomes isolated for egress, and all unicast egress traffic is blocked."